config.yml

```diff
@@ -133,3 +125,9 @@ docker:
   scenario_geographies_list: ["Global", "NonOECD", "OECD"]
   global_aggregate_scenario_sources_list: ["WEO2022"]
   global_aggregate_sector_list: ["Power"]
+
+2022Q4_docker:
+  inherits: 2022Q4
+  data_prep_outputs_path: "/mnt/outputs"
+  asset_impact_data_path: "/mnt/asset-impact"
+  factset_data_path: "/mnt/factset-extracted/factset-pacta_timestamp-20221231T000000Z_pulled-20240217T134528Z"
```

Comment:
These require a change to `docker-compose.yml`. Maybe that's in another PR, but I'd rather not merge PRs that depend on another PR until the other PR is merged. So I guess either this PR needs to wait, or it needs to import the change here.

Reply:
in order to keep PRs focussed, I'd prefer that this PR wait (until the necessary infrastructure is already merged)

Reply:
Added changes to `docker-compose.yml` to this PR.